controlplane/balancer.h

```diff
@@ -26,9 +26,11 @@ using service_key_t = std::tuple<common::ip_address_t,
 using service_counter_key_t = std::tuple<std::string, ///< module_name
                                          service_key_t>;
 
-using real_counter_key_t = std::tuple<std::string, ///< module_name
-                                      service_key_t,
-                                      real_key_t>;
+using real_key_global_t = std::tuple<std::string, ///< module_name
+                                    service_key_t,
+                                    real_key_t>;
```

Comment:
Is `tuple` really needed here? We should prefer `struct` to `tuple` in cases where we don't need to iterate over members.

If resorting to using tuple is unavoidable generous usage of type aliases makes code more readable:
```
using ModuleName = std::string
std::tuple<ModuleName>
```
is better than
```
std::tuple<std::string ///> module_name
>
```